docs/blog_post.md

```diff
@@ -0,0 +1,56 @@
+
+Fortunately, it is now possible to create computer programs that label images with accuracy comparable to that of humans. Moreover, ShopRunner's data science team has experience building such programs to classify products across ShopRunner's retail network for its marketplace app District. For over two years, **data scientists from ShopRunner and Uptake have been working with the Urban Wildlife Institute to use automated image classification techniques to unblock the institute's research efforts.** 
+
+**The data and code used for this project are [publicly available](https://github.com/uptake/autofocus) so that anyone interested in developing the relevant skills can get involved.**
+
+## Impact
+
+> It was like they gifted us 10 interns who could work around the clock. Autofocus enables us to get on top of our backlog of images and deliver valuable insights on local wildlife now.
+>
+> --Urban Wildlife Institute researcher Mason Fidino, PhD <!--TK image of Mason-->
+
+Approximately 10-15 percent of the images that the Urban Wildlife Institute's cameras capture contain humans rather than the wildlife that is relevant to the institute's research. Those images are a major nuisance because the institute wishes to respect privacy by filtering out those images before posting the rest online for volunteers to label.
+
+![](human_example.JPG)
+<!--TK image caption formatting-->
+The Urban Wildlife Institute's cameras often capture images of people jogging or walking dogs, or (as in this case) images of the researchers themselves stopping by to service them.
+
+To address this problem, Uptake's philanthropic arm Uptake.org developed the first version of a program called Autofocus that can classify an image as containing a human or not. That program proved to be sufficiently accurate to remove approximately 50% of human-containing images with a negligible rate of false positives. **On one batch of images from Spring 2016, the Urban Wildlife Institude was able to use that model to filter out roughly 3,700 images.** "It was like they gifted us 10 interns who could work around the clock," said Urban Wildlife Institute researcher Mason Fidino. "Autofocus enables us to get on top of our backlog of images and deliver valuable insights on local wildlife now."
+
+**Data scientists at ShopRunner contributed to a second version of Autofocus that not only appears to increase human label accuracy, but also provides labels for eighteen additional kinds of animals.** The accuracy of that second version is currently under review, but preliminary results suggest that it is sufficient to automate classification of a large percentage of incoming images at least for common animal types such as deer and raccoons, further improving the speed of the Urban Wildlife Institute's research.
```

Comment:
Could you currently give _any_ concrete metrics on performance?

Reply:
There are some in https://github.com/uptake/autofocus/blob/master/autofocus/train_model/train_multilabel_model.ipynb. I'm not sure I have anything offhand that I could report with little explanation without being confusing or potentially misleading. E.g. I have an accuracy score, but it is a bit confusing in that it's aggregated over all categories, and there are some subtleties related to the scope of the dataset and the way the train/test splits were done that are hard to convey briefly. Mason is working to run the model on a more representative set of images and to tune the thresholds per-category, which will then allow us to say things about e.g. how many images they can remove with an acceptably low false-positive rate.